.gitpod.yml

```
@@ -0,0 +1,11 @@
+# Commands to start on workspace startup
+tasks:
+  - name: Dev Server
+    init: npm install
```

@chalin, should this be:

```suggestion
    init: npm install --lts
```

---

Yes